### PR TITLE
feat: Add linspace examples

### LINKED ISSUE #7553: linspace

### PR DESCRIPTION
**Pull request by meetcshah19**
Close #7553

### CODE REVIEW

**Comment by nassimberrada** [comment]
Hi @meetcshah19, thank you for your contribution ! 
The functional examples look good to me but could you add container examples as well ? Thanks

**Comment by meetcshah19** [comment]
Sure, on it!

**Comment by meetcshah19** [comment]
@nassimberrada are you sure linspace() can actually be used with Containers is a way that would be logically required/ plausible? If yes, can you give me an example of such a case? To me applying space on a nested array structure doesn't make a lot of sense. Thanks!

**Comment by nassimberrada** [comment]
Also, i just saw that a new PR overlaps with yours, i think this was caused by improper naming, could you rename your PR with just the function name ? Feel free to follow the same logic for future PRs as well to avoid any confusion.

**Comment by meetcshah19** [comment]
Thanks for the explanation @nassimberrada, I did mean a real life example. Also, I get the mixup with the issue tracking and I believe the solution you suggested is ideal. I will write the array instance method example.

File: ivy/functional/ivy/creation.py [hunk 1]
```diff
@@ -1076,14 +1076,24 @@ def linspace(
...
-    ivy.array([[1, 2, 3, 4],[2, 3, 4, 5]])
+    ivy.array([[1, 2, 3, 4], [2, 3, 4, 5]])
 
     >>> ivy.linspace(x, y, 4, axis = 0)
     ivy.array([[1, 2],
                [2, 3],
                [3, 4],
                [4, 5]])
 
+    Using array instance method
```
Comment (nassimberrada) [inline ivy/functional/ivy/creation.py:1087]: Could you add/change just the following line(s):
```
    Instance Method Examples
    -------------------
    With :class:`ivy.Array` input:
```

**Review by nassimberrada — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/creation.py (hunk 1 above)
Comment (meetcshah19) [reply ivy/functional/ivy/creation.py]: Done

**Comment by nassimberrada** [comment]
There does seem to be an issue with the docstrings tests since your example output is correct. 
Fixing the testing process should make your examples pass but since that is not your responsibility here, i can merge the PR as it looks good to me. 
Thank you for your patience, and your contribution to Ivy !

**Pull request merged**